lib/helpers.coffee

```diff
@@ -1,8 +1,35 @@
 SelectorLinter = require 'atom-selector-linter'
+CSON = require 'season'
+fs = require 'fs-plus'
 
 exports.getSelectorDeprecations = ->
   linter = new SelectorLinter(maxPerPackage: 50)
   linter.checkPackage(pkg) for pkg in atom.packages.getLoadedPackages()
+  
+  userKeymapPath = atom.keymaps.getUserKeymapPath()
+  
+  if fs.existsSync(userKeymapPath)
+    linter.checkKeymap(CSON.readFileSync(atom.keymaps.getUserKeymapPath()), {
+      packageName: "your local #{path.basename(userKeymapPath)} file",
+      packagePath: "",
+      sourcePath: userKeymapPath
+    })
+    
+  userStyleSheetPath = atom.styles.getUserStyleSheetPath()
+  
+  if fs.existsSync(userStyleSheetPath)
+    userStyleSheet = fs.readFileSync(atom.styles.getUserStyleSheetPath(), 'utf8')
+    linter.checkSyntaxStylesheet(userStyleSheet, {
+      packageName: "your local #{path.basename(userStyleSheetPath)} file",
+      packagePath: "",
+      sourcePath: userStyleSheetPath
+    })
+    linter.checkUIStylesheet(userStyleSheet, {
```

Comment:
I'm not 100% sure on this but I think both check might be needed, perhaps @nathansobo knows the answer here.